packages/cli/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../typescript/tsconfig.library.json",
   "compilerOptions": {
     "rootDir": "src",
-    "outDir": "lib"
+    "outDir": "lib",
+    "esModuleInterop": true
```

Comment:
required to get the CLI working, see https://github.com/celo-org/celo-monorepo/pull/795